### ISSUE TITLE
Issues with the engine crane or something idk 

### ISSUE THREAD

**Issue by Lyricalsteve** [zephyrj/engine-crane#38]
This is the latest version I can’t get any cars that I made in automation to work at all this is what is says on engine crane 

It says failed to load automation car file stream doesn’t open with expected blob mark is it valid .cardata ?.

have tried everything from reinstalling beamNG and automation but still nothing

**Comment by zephyrj** [comment]
Are you using the new Al-Rima update?
https://github.com/zephyrj/engine-crane/issues/39

**Comment by Lyricalsteve** [comment]
Yes I am

**Comment by zephyrj** [comment]
I'll look into updating it to work with the new BeamNG and car version - in the meantime I did a quick test and the custom exporter I wrote seems to (miraculously) still work.
Details about that are here: [https://youtu.be/4Xxbg2GUnqc](https://www.youtube.com/watch?v=4Xxbg2GUnqc)

That should allow you to still do engine swaps. I'll track the fixing of this under the other issue #39 and close this one for now.
